tests/conftest.py

```diff
@@ -13,7 +13,16 @@
 import pytest
 
 # cisagov Libraries
-from models.models import SMTP, Assessment, Campaign, Group, Page, Target, Template
+from models.models import (
+    SMTP,
+    Assessment,
+    Campaign,
+    Click,
+    Group,
+    Page,
+    Target,
+    Template,
```

Comment:
We always prefer to alphabetize wherever possible:

```suggestion
    Assessment,
    Campaign,
    Click,
    Group,
    Page,
    SMTP,
    Target,
    Template,
```

Reply:
@JCantu248 - Reminder that this item still has not been addressed.

Reply:
@JCantu248 - Pinging again on this small alphabetization issue.

Reply:
It looks like isort in pre-commit will complain with the requested changes to the import list and revert them back to the original way.

Reply:
That is strange, but if that is what `isort` is doing, we can resolve this for now.

@mcdonnnj this oddness is something that you might be interested in looking into at some point in the future.